Signal/src/ViewControllers/DebugUITableViewController.m

```diff
@@ -113,11 +113,1189 @@ + (void)presentDebugUIForThread:(TSThread *)thread
+        @"Bonner",
+        @"Cotton",
+        @"Holder",
+        @"Stark",
+        @"Ferrell",
+        @"Cantrell",
+        @"Fulton",
+        @"Lynn",
+        @"Lott",
+        @"Calderon",
+        @"Rosa",
+        @"Pollard",
+        @"Hooper",
+        @"Burch",
+        @"Mullen",
+        @"Fry",
+        @"Riddle",
+        @"Levy",
+        @"David",
+        @"Duke",
+        @"O'donnell",
+        @"Guy",
+        @"Michael",
+        @"Britt",
+        @"Frederick",
+        @"Daugherty",
+        @"Berger",
+        @"Dillard",
+        @"Alston",
+        @"Jarvis",
+        @"Frye",
+        @"Riggs",
+        @"Chaney",
+        @"Odom",
+        @"Duffy",
+        @"Fitzpatrick",
+        @"Valenzuela",
+        @"Merrill",
+        @"Mayer",
+        @"Alford",
+        @"Mcpherson",
+        @"Acevedo",
+        @"Donovan",
+        @"Barrera",
+        @"Albert",
+        @"Cote",
+        @"Reilly",
+        @"Compton",
+        @"Raymond",
+        @"Mooney",
+        @"Mcgowan",
+        @"Craft",
+        @"Cleveland",
+        @"Clemons",
+        @"Wynn",
+        @"Nielsen",
+        @"Baird",
+        @"Stanton",
+        @"Snider",
+        @"Rosales",
+        @"Bright",
+        @"Witt",
+        @"Stuart",
+        @"Hays",
+        @"Holden",
+        @"Rutledge",
+        @"Kinney",
+        @"Clements",
+        @"Castaneda",
+        @"Slater",
+        @"Hahn",
+        @"Emerson",
+        @"Conrad",
+        @"Burks",
+        @"Delaney",
+        @"Pate",
+        @"Lancaster",
+        @"Sweet",
+        @"Justice",
+        @"Tyson",
+        @"Sharpe",
+        @"Whitfield",
+        @"Talley",
+        @"Macias",
+        @"Irwin",
+        @"Burris",
+        @"Ratliff",
+        @"Mccray",
+        @"Madden",
+        @"Kaufman",
+        @"Beach",
+        @"Goff",
+        @"Cash",
+        @"Bolton",
+        @"Mcfadden",
+        @"Levine",
+        @"Good",
+        @"Byers",
+        @"Kirkland",
+        @"Kidd",
+        @"Workman",
+        @"Carney",
+        @"Dale",
+        @"Mcleod",
+        @"Holcomb",
+        @"England",
+        @"Finch",
+        @"Head",
+        @"Burt",
+        @"Hendrix",
+        @"Sosa",
+        @"Haney",
+        @"Franks",
+        @"Sargent",
+        @"Nieves",
+        @"Downs",
+        @"Rasmussen",
+        @"Bird",
+        @"Hewitt",
+        @"Lindsay",
+        @"Le",
+        @"Foreman",
+        @"Valencia",
+        @"O'neil",
+        @"Delacruz",
+        @"Vinson",
+        @"Dejesus",
+        @"Hyde",
+        @"Forbes",
+        @"Gilliam",
+        @"Guthrie",
+        @"Wooten",
+        @"Huber",
+        @"Barlow",
+        @"Boyle",
+        @"Mcmahon",
+        @"Buckner",
+        @"Rocha",
+        @"Puckett",
+        @"Langley",
+        @"Knowles",
+        @"Cooke",
+        @"Velazquez",
+        @"Whitley",
+        @"Noel",
+        @"Vang",
+    ];
+    return values[arc4random() % values.count];
```

Comment:
Not that it matters in this context, but it's a good habit to avoid modulo bias by using arc4random_uniform (and it's slightly more ergonomic IMO). 

```
return values[arc4random_uniform(values.count)]
```